ui/lib/core/addon/modifiers/code-mirror.js

```diff
@@ -10,6 +10,7 @@ import Modifier from 'ember-modifier';
 
 import 'codemirror/addon/edit/matchbrackets';
 import 'codemirror/addon/selection/active-line';
+import 'codemirror/addon/display/autorefresh';
```

Comment:
[[docs](https://codemirror.net/5/doc/manual.html#addon_autorefresh]

When a modal renders the `JsonEditor` (ex: with the sample policy template) the content of JsonEditor doesn't render unless the element is focused. Previously we would wait to render the component by wrapping the element in an additional conditional, but this approach no longer works. Now codemirror will autorefresh if passed a `@container` arg

https://github.com/hashicorp/vault/blob/main/ui/app/templates/components/oidc/scope-form.hbs#L100-L104

Reply:
Do you think it would be necessary in scenarios other than in a modal where the container value is passed to set the `autoRefresh` property? I'm guessing no since we haven't run into this until now and we could easily add a dedicated arg for `autoRefresh` in the future if needed.

Reply:
Yeah - that was my thinking. For now, it seemed okay to have autoRefresh rely on container. But since container could refer to a flyout, dropdown, or maybe even a card - we may want to separate the args in the future!